presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeProbabilityFunctionQueries.java

```diff
@@ -90,4 +90,13 @@ public void testNormalCDF()
         assertQuery("SELECT normal_cdf(9.15, nationKey, 5.3) FROM nation");
         assertQuery("SELECT normal_cdf(2.3, 11.2, nationKey) FROM nation");
     }
+
+    @Test
+    public void testWilsonInterval()
+    {
+        assertQuery("SELECT wilson_interval_upper(3, 7, acctbal / 200) FROM supplier WHERE acctbal > 0.0 AND acctbal < 1000.0");
```

Comment:
Just curious why you have all the predicates that check > 0 (or 0.0) ? Wouldn't those predicates always be true in the dataset ?

Reply:
You are right, these predicates should always be true. However, I felt the code would be slightly more robust if I don't make more assumptions than are absolutely required

Reply:
Many tests seem to have these predicates because the underlying parameters requires particular (say >0) values and so these filters are ensuring that.

In general, the expectation is that queries should look representative of the end use-case.

Reply:
Turns out that these predicates are NOT always true in the dataset. For example:
```
presto> select acctbal from tpch.tiny.supplier where acctbal < 0;
 acctbal 
---------
 -283.84 
 -224.84 
  -40.45 
 -724.31 
 -963.79 
 -632.16 
 -831.07 
 -290.06 
  -966.2 
 -891.99 
 -811.62 
(11 rows)
```
In this case, the predicates need to be there.

Reply:
Thanks for confirming !